examples/rebalancer/src/requests/createActivityApproval.ts

```diff
@@ -1,35 +1,34 @@
-import { TurnkeyApi, init as httpInit, withAsyncPolling } from "@turnkey/http";
+import { TurnkeyClient } from "@turnkey/http";
+import { createActivityPoller } from "@turnkey/http/dist/async";
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
 import { TurnkeyActivityError } from "@turnkey/ethers";
 import { refineNonNull } from "./utils";
 
 export default async function approveActivity(
   activityId: string,
   activityFingerprint: string
 ): Promise<string> {
```

Comment:
Note: for all these files (`in src/requests/*.ts`), I think we should pass the TurnkeyClient as an argument potentially? That'd avoid having to read process.env in every file.

Not a big deal though, it's an example; not a production-ready app!